astroquery/vo_conesearch/core.py

```diff
@@ -30,8 +30,8 @@ class ConeSearchClass(BaseQuery):
     The class for querying the Virtual Observatory (VO)
     Cone Search web service.
 
-    Example
-    -------
+    Examples
```

Comment:
yep, new sphinx issues a tons of warning, this was one of those (though it didn't make the test fail unlike the other one)